packages/stk/stk_balance/stk_balance/junk2.1_to_2.log

```diff
@@ -0,0 +1,47 @@
+stk_balance converting from 1 to 2 MPI ranks
```

Comment:
What is this log file for?

Reply:
Good catch I hadn't noticed that, that's an accidental file add somehow. I'll make sure it is removed in sierra so it won't show up in future snapshots.

Reply:
So that file is not checked into the sierra repo. Our snapshot script makes sure we are snapshotting from a clean clone of sierra, but since that log file was ignored by git (due to a .gitignore file), the snapshot script didn't 'see' it. But then the file got brought over to trilinos by a 'cp -r' command. So I guess we just need to make sure our source directory is really clean before snapshotting.